modes/passenger-services.md

```diff
@@ -0,0 +1,204 @@
+# Mobility Data Specification: **Passenger Services**
+
+<img src="https://i.imgur.com/plW2Hon.png" width="120" align="right" alt="MDS Modes - Passenger Services" border="0">
+
+**Passenger Services** refers to taxis and transportation network companies (TNCs).  Passenger Services typically have a driver, one or more passengers, and multiple passengers may be on different trips.  The state machine tracks the trip states of the passengers separately from the vehicle state.  
+
+See the [modes overview](/modes) for how the mode specific information below applies across MDS.
+
+## Taxi vs. TNC implementation differences
+
+Taxis typically require explicit tracking of maintenance while TNCs typically do not.
+
+## Table of Contents
+
+- [Mode Attributes](#mode-attributes)
+- [Vehicle States](#vehicle-states)
+- [Event Types](#event-types)
+- [Vehicle State Events](#vehicle-states-events)
+- [State Machine Diagram](#state-machine-diagram)
+
+## Mode Attributes
+
+### Mode ID
+
+The short name identifier for Passenger Services used across MDS is `passenger-services`.
+
+### Journey ID
+
+The `journey_id` field shall have a consistent value in overlapping trips, e.g. "pooled" or "shared" rides with different start and/or end locations.
+
+### Trip ID Requirements
+
+Events require a valid `trip_id` in events where `event_types` contains `reservation_start`, `reservation_stop`, `trip_start`, `trip_stop`, `trip_end`, `passenger_cancellation`, `provider_cancellation`, or `driver_cancellation`. 
+
+Additionally, `trip_id` is required if `event_types` contains a `enter_jurisdiction` or `leave_jurisdiction` event pertaining to a passenger trip. 
+
+### Trip Type
+
+The `trip_type` field is not currently used in passenger services.
+
+### Trip Attributes
+
+The `trip_attributes` array is not used with in passenger services.
+
+### Vehicle Attributes
+
+The `vehicle_attributes` array may have the following key value pairs:
```

Comment:
Should there be other attributes defined here, per the [modes planning doc](https://docs.google.com/document/d/1lzQl_EvrpiDWcJidmnKhn6bAlOdimEBo5u_Yx0BTfmk/edit#heading=h.6m74qj1b4uv1), like `passenger_capacity` or `wheelchair_accessible`, etc?